README.md

```diff
@@ -143,10 +143,12 @@ that required paid sponsorship upon exceeding a free limit.
 
 ### `config` stage
 
-| Argument       | Default | Description                                             |
-|----------------|---------|---------------------------------------------------------|
-| `--builds-dir` |         | Path to a directory on host to use for storing builds   |
-| `--cache-dir`  |         | Path to a directory on host to use for caching purposes |
+| Argument             | Default | Description                                              |
+|----------------------|---------|----------------------------------------------------------|
+| `--builds-dir`       |         | Path to a directory on host to use for storing builds    |
+| `--cache-dir`        |         | Path to a directory on host to use for caching purposes  |
+| `--guest-builds-dir` |         | Path to a directory in guest to use for storing builds   |
+| `--guest-cache-dir`  |         | Path to a directory in guest to use for caching purposes |
```

Comment:
From reading this documentation it's not clear to me why we need `--guest-*` and how to set it. They are only useful when we mount a disk or a block device, right?

These options seems to mutual exclusive and both use cases should be explained.